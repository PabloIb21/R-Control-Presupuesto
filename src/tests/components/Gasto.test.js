import { mount } from 'enzyme';
import React from 'react';
import Gasto from '../../components/Gasto';

describe('<Gasto />', () => {
  test('mount', () => {
    const gasto = {
      nombre: 'Transporte',
      cantidad: 300
    };

    const wrapper = mount(
      <Gasto gasto={gasto} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});