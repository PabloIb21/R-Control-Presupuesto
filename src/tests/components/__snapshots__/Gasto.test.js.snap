// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Gasto /> mount 1`] = `
<li
  className="gastos"
>
  <p>
    Transporte
    <span
      className="gasto"
    >
      $ 
      300
    </span>
  </p>
</li>
`;
